### ISSUE TITLE
g_loss, d1_loss, d2_loss

### ISSUE THREAD

**Issue by Mrz12121** [Niwhskal/SRNet#6]
I wanna know what your values of g_loss, d1_loss, d2_loss are

**Comment by Niwhskal** [comment]
Hello Mrz12121 ,

g_loss = 10.01
d1_loss = 14.8
d2_loss = 17.1

Look for a generator loss of 9-12. Ignore the discriminator losses.
